app/controllers/application_controller.rb

```diff
@@ -28,7 +33,20 @@ def link_notice_and_redirect(notice, link_path, link_text, redirect_url)
     redirect_to redirect_url
   end
 
+  def user_not_authorized
+    flash[:alert] = I18n.t :unauthorized_logout
+    sign_out current_user
+    render :unauthorized, status: :unauthorized, layout: false
+  end
+
   add_flash_types :undo_notice, :link_notice
 
   helper_method :session_path
+
+  private
+
+  # https://github.com/charliesome/better_errors/issues/341
```

Comment:
Could use a quick blurb here in case the URL dies or... I'm lazy

Reply:
Yup fair point.